pkg/reconciler/pipelinerun/resources/apply_test.go

```diff
@@ -2034,6 +2034,285 @@ func TestApplyParameters_ArrayIndexing(t *testing.T) {
 	}
 }
 
+func TestApplyReplacementsMatrix(t *testing.T) {
+	ctx := context.Background()
+	cfg := config.FromContextOrDefaults(ctx)
+	cfg.FeatureFlags.EnableAPIFields = config.BetaAPIFields
```

Comment:
why are we enabling beta?

Reply:
Oops, this was copied. Changed to EnableAlphaAPIFields